plugins/addon/controller.go

```diff
@@ -54,6 +55,7 @@ var _ reconcile.Reconciler = &{{ .Resource.Kind }}Reconciler{}
 type {{ .Resource.Kind }}Reconciler struct {
 	client.Client
 	Log logr.Logger
+	Scheme *runtime.Scheme
 
 	declarative.Reconciler
```

Comment:
it was done here: 

https://github.com/kubernetes-sigs/kubebuilder/pull/1162/files#diff-facd64112f41d2a3cfc18d60b19a08f5R45-R58

+ docs updates.